src/component/Home.js

```diff
@@ -3,11 +3,13 @@ import GetToken from './GetToken';
 
 function Home() {
   return (
-    <div>
-      <h1>New Shopping List</h1>
-      <GetToken />
-      <p><a href="/join">Join Existing List</a></p>
-    </div>
+    <React.Fragment>
```

Comment:
You only need to use <React.Fragment> when you would otherwise have multiple nodes at the top level of the component. In this case, you have a <div> that can act as that single top-level component, so you can safely remove the Fragment tag.